.github/workflows/ci.yaml

```diff
@@ -0,0 +1,159 @@
+name: CI
+
+on: push
```

Comment:
All of these CI workflows should only happen on `pull_request`. That's the [typical setting](https://app.circleci.com/settings/project/github/zendeskgarden/react-containers/advanced?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fzendeskgarden%2Freact-containers) for all CircleCI repos. We just turned it off temporarily on `react-components` because there was no way to get `next` to build in addition to `main`.

Reply:
Does this config/GHA now allow us to only build on PR for select branches? 🤞🏻

Reply:
Yes, it supports it.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-the-head-or-base-branch-of-a-pull-request

```
on:
  pull_request:
    branches:
      - 'releases/**'
```